source/blog/2020-09-02-solidus-paypal-commerce-platform.html.markdown

```diff
@@ -57,6 +57,10 @@ When a user clicks the PayPal button, Solidus sends line_item, adjustment, shipp
 
 When a user logs in or changes their address on PayPal, PayPal sends a partial shipping address (missing only street address) and expects us to return an estimate for shipping. We simulate this change in the shipping rates controller, and return what the shipping would be for that address. Then, after the user confirms their payment, we get the finalized address from PayPal. This shouldn't cause you any issues unless you have a lot of customization around shipping rates, but you should keep it in mind just in case.
 
+### Referral Fee
+
+By using this extension, you are giving back to Solidus. PayPal will pay a fee to the maintainers ([Nebulab](https://nebulab.it)) for each order made through the payment gateway.
```

Comment:
I want to be 100% transparent here, and technically you are _not_ giving back to Solidus. Can we remove that? Here's what we could replace it with instead:

> This extension has been sponsored by PayPal and implemented by Nebulab. PayPal will pay a commission fee to the maintainers for each order placed through the new payment gateway.

Reply:
Makes sense! I've updated it. Thanks!